internal/cli/auth/login.go

```diff
@@ -226,14 +226,22 @@ func (opts *LoginOpts) PreRun() error {
 	return opts.initFlow()
 }
 
+func Tool() string {
+	if config.ToolName == config.MongoCLI {
+		return "Ops Manager or Cloud Manager"
```

Comment:
```suggestion
		return "Atlas or Cloud Manager"
```
Ops manager is not supported, atlas is still supported

Reply:
Updated, thank you! -> I'll return Cloud Manager if it's mongo cli and Atlas if it is atlas cli so there will be no "or " here.